config/networks/mainnet/config.yaml

```diff
@@ -96,5 +88,29 @@ DEPOSIT_CONTRACT_ADDRESS: 0x00000000219ab540356cBB839Cbe05303d7705Fa
 
 # Networking
 # ---------------------------------------------------------------
+# `10 * 2**20` (= 10485760, 10 MiB)
+GOSSIP_MAX_SIZE: 10485760
```

Comment:
where did you get this stuff from?

Reply:
https://github.com/ethereum/consensus-specs/blob/dev/configs/mainnet.yaml
https://github.com/ethereum/consensus-specs/blob/dev/configs/minimal.yaml